libraries/models.py

```diff
@@ -378,6 +378,12 @@ class LibraryVersion(models.Model):
     deletions = models.IntegerField(default=0)
     files_changed = models.IntegerField(default=0)
     cpp_standard_minimum = models.CharField(max_length=50, blank=True, null=True)
+    dependencies = models.ManyToManyField(
+        "Library",
```

Comment:
Above, the FK to library uses `libraries.Library`. Doesn't really matter, but we should probably be consistent.